server/services/messaging_service_test.go

```diff
@@ -1495,20 +1495,32 @@ func TestConnect(t *testing.T) {
 	}
 	s := NewMessagingServer(&mockIdentityServer{})
 
+	errs := make(chan error, 1)
```

Comment:
This is fine, but there is also [errgroup](https://pkg.go.dev/golang.org/x/sync/errgroup)

Reply:
That's way better, I'll do that, thanks!

Reply:
Done.